src/app/components/shell/router-helpers/fallback-to-general.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Error404 from '~/pages/404/404';
+import {useToggle} from '~/helpers/data';
+import {GeneralPageFromSlug} from '~/pages/general/general';
+
+export default function FallbackToGeneralPage({name}) {
+    const [fallback, setFallback] = useToggle(false);
+
+    if (fallback) {
+        return <Error404 />;
+    }
+
+    return (
+        <GeneralPageFromSlug slug={`spike/${name}`} fallback={setFallback} />
+    );
```

Comment:
some of these components might read a little more concisely if you used ternaries rather than if statements, but that's just a personal preference